.gitignore

```diff
@@ -1,11 +1,17 @@
 .eggs/
 .DS_Store
+venv3.7/
```

Comment:
I would probably not include this in the `.gitignore` file: it's not a standard name nor is it created by anything automatically; we don't want this file to grow to include everyone's individual preferences for virtual environment names.  You can use the file `.git/info/exclude` as a local `.gitignore` that isn't part of the repository.